pkg/types/powervs/powervs_regions.go

```diff
@@ -0,0 +1,77 @@
+package powervs
+
+// Since there is no API to query these, we have to hard-code them here.
+
+// Region describes resources associated with a region in Power VS.
+// We're using a few items from the IBM Cloud VPC offering. The region names
+// for VPC are different so another function of this is to correlate those.
+type Region struct {
+	Name        string
+	Description string
+	VPCRegion   string
+	Zones       []string
+}
+
+// Regions holds the regions for IBM Power VS, and descriptions used during the survey
+var Regions = map[string]Region{
+	"dal": {
+		Name:        "dal",
+		Description: "Dallas, USA",
+		VPCRegion:   "us-south",
+		Zones:       []string{"dal12"},
+	},
+	"eu-de": {
+		Name:        "eu-de",
+		Description: "Frankfurt, Germany",
+		VPCRegion:   "eu-de",
+		Zones: []string{
+			"eu-de-1",
+			"eu-de-2",
+		},
+	},
+	"lon": {
+		Name:        "lon",
+		Description: "London, UK.",
+		VPCRegion:   "eu-gb",
+		Zones: []string{
+			"lon04",
+			"lon06",
+		},
+	},
+	"osa": {
+		Name:        "osa",
+		Description: "Osaka, Japan",
+		VPCRegion:   "jp-osa",
+		Zones:       []string{"osa21"},
+	},
+	"syd": {
+		Name:        "syd",
```

Comment:
the key itself is the name. do you want to include a duplicate name field as well?

Reply:
nope, we can remove that `Name` field from `Region`.